lib/open_api_spex/plug/swagger_ui.ex

```diff
@@ -192,10 +192,16 @@ defmodule OpenApiSpex.Plug.SwaggerUI do
     end
   end
 
-  defp supplement_config(%{oauth2_redirect_url: {:endpoint_url, path}} = config, conn) do
-    endpoint_module = Phoenix.Controller.endpoint_module(conn)
-    url = Path.join(endpoint_module.url(), path)
-    Map.put(config, :oauth2_redirect_url, url)
+  case Code.ensure_compiled(Phoenix.Controller) do
```

Comment:
Just ensure_loaded is enough?

Reply:
@wingyplus Good point, according to the [docs](https://hexdocs.pm/elixir/1.13/Code.html#module-code-loading-on-the-erlang-vm) `ensure_loaded/1` should be preferred. Updated.